CDP4Addin.Tests/CDP4Addin.Tests.csproj

```diff
@@ -7,6 +7,7 @@
     <Product>CDP4Addin.Tests</Product>
     <Description>CDP4Addin Unit tests</Description>
     <Copyright>Copyright © Starion Group S.A.</Copyright>
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
does this mean we can no longer support 32 bit excel?

Reply:
Forgot to reset it to AnyCPU, will fix